spec/models/manageiq/providers/openstack/cloud_manager/provision_workflow_spec.rb

```diff
@@ -298,24 +301,51 @@
 
         context "floating ip" do
           before do
-            @ip1 = FactoryGirl.create(:floating_ip, :address => "1.1.1.1")
-            @ip2 = FactoryGirl.create(:floating_ip, :address => "2.2.2.2")
-            provider.network_manager.floating_ips << @ip1
-            provider.network_manager.floating_ips << @ip2
-            @ct1.floating_ips << @ip1
-            @ct2.floating_ips << @ip2
+            cloud_network_public   = FactoryGirl.create(:cloud_network_openstack_public)
+            cloud_network_public_2 = FactoryGirl.create(:cloud_network_openstack_public)
+            router                 = FactoryGirl.create(:network_router_openstack,
+                                                        :cloud_network => cloud_network_public)
+            @cloud_network         = FactoryGirl.create(:cloud_network_openstack_private,
+                                                        :cloud_tenant => @ct2)
+            @cloud_network_2       = FactoryGirl.create(:cloud_network_openstack_private,
+                                                        :cloud_tenant => @ct2)
+            _subnet                = FactoryGirl.create(:cloud_subnet_openstack,
```

Comment:
@syncrou The variable, no, there is a _ so rubocop doesn't complain. But it needs to be there, since the private network connects to router through subnets.